docs/content/glossary.md

```diff
@@ -1,28 +1,46 @@
 # Glossary
 
-**SpatialOS terms**<br/>
```

Comment:
Changed all the glossary entries so that they had headings (so we can link directly to them from other files).